documentation/projects.md

```diff
@@ -100,6 +103,28 @@ MODULEPATH:
 Use `bolt plan show <plan-name>` to view details and parameters for a specific plan.
 ```
 
+You can also use glob patterns to match multiple plan or task names. For
+example, to surface all tasks and plans in a project named `myproject`, you
+would use the following `bolt-project.yaml` file:
+
+```yaml
+name: myproject
+plans:
+- myproject::*
+tasks:
+- myproject::*
+```
+
+The following metacharacters can be used in a glob pattern:
+
+| Metacharacter | Description |
+| --- | --- |
+| `*` | Matches any number of characters. |
+| `?` | Matches any one character. |
+| `[set]` | Matches any one character in the set. |
+| `\` | Escapes the next metacharacter. |
```

Comment:
I don't think this is necessary since none of the metacharacters are valid parts of a modulename anyway.